app/email.py

```diff
@@ -73,4 +74,21 @@ def register(form, role):
 def resend_conf(user):
     confirm_url = get_confirm_url(user)
     send_conf_email(user, confirm_url)
-    flash('A new confirmation email has been sent.', 'success')
\ No newline at end of file
+    flash('A new confirmation email has been sent.', 'success')
+
+
+def add_user():
```

Comment:
What is this function and the function below it [remove_users()] for?

Reply:
Testing.